oap-server/server-storage-plugin/storage-iotdb-plugin/pom.xml

```diff
@@ -31,8 +31,7 @@
     <packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <iotdb-session.version>0.12.2</iotdb-session.version>
```

Comment:
IoTDB client doesn't require JDK 11, right? SkyWalking is widely used in JDK8 for now.

Reply:
IoTDB client needs JDK>=1.8